src/design-system/styles/initThemingLocal.ts

```diff
@@ -0,0 +1,9 @@
+import { currentThemeStore } from '~/core/state/currentSettings/currentTheme';
+
+export function initThemingLocal() {
+  const localTheme = localStorage.getItem('theme');
+  if (!localTheme) {
+    const currentTheme = currentThemeStore.getState().currentTheme;
```

Comment:
is the extra localStorage set/get needed? can't just use the zustand store like this?

Reply:
guessing this is so the static vanilla js scripts have somewhere to read from

Reply:
yeah exactly what @derHowie says

Reply:
aah zustand is using an async `chrome.storage.local` for everything, maybe we could consider opting for standard `localStorage` when fit
so the store is already hydrated when created vs having to wait for the async run to hydrate it